requirements.txt

```diff
@@ -15,3 +15,4 @@ Flask-SQLAlchemy==2.4.1
 Flask-DebugToolbar==0.10.1
 PyMySQL==0.9.3
 cryptography==2.7
+sphinx==2.2.0
```

Comment:
this is only preliminary since the webapp code doesn't actually depend on sphinx being installed

Reply:
I checked the RTD docs and the only way to specify (and lock) a particular version was by specifying it in the reqs.txt

Source: https://docs.readthedocs.io/en/stable/intro/import-guide.html#building-your-documentation

Reply:
Are there any problems letting RTD use Sphinx v1.8.5?

Reply:
Not particularly, but since v1 to v2 indicates a major change in the API, it's safer to start out with the latest version. Also, if there are feature releases in v2 that can help us, it's safer to just go from 2.2.0 to 2.3.0, rather than 1.8.5 to 2.3.0